### PR TITLE
Sponsor page

### PR DESCRIPTION
**Pull request by KhushmeetChugh**
https://github.com/technobyte-nitkkr/Spardha/assets/133222681/1d4def58-5420-4416-81c4-86fda9719fca

Responsiveness added to sponsor page

### CODE REVIEW

File: apps/docs/app/globals.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 :root {
-  --max-width: 1100px;
+  --max-width: 100px;
```
Comment (techsavvyash) [inline apps/docs/app/globals.css:2]: why this?

File: apps/web/app/globals.css [hunk 2]
```diff
@@ -67,7 +67,7 @@
 }
 
 .CardScrollDiv::-webkit-scrollbar-thumb {
-  background-color: #3182ce;
+  background-color: #ce7831;
```
Comment (techsavvyash) [inline apps/web/app/globals.css:70]: why this?

File: apps/web/app/page.module.css [hunk 3]
```diff
@@ -4,9 +4,8 @@
   justify-content: space-between;
   align-items: center;
   padding: 6rem;
-  min-height: 100vh;
+  min-height: 10000vh;
```
Comment (techsavvyash) [inline apps/web/app/page.module.css:7]: why this?

File: apps/docs/app/globals.css (hunk 1 above)
Comment (KhushmeetChugh) [reply apps/docs/app/globals.css:2]: sir check again updated

**Pull request closed without merging**